Sources/CocoaLumberjack/Extensions/DDDispatchQueueLogFormatter.m

```diff
@@ -209,7 +221,7 @@ - (NSString *)formatLogMessage:(DDLogMessage *)logMessage {
     NSString *timestamp = [self stringFromDate:(logMessage->_timestamp)];
     NSString *queueThreadLabel = [self queueThreadLabelForLogMessage:logMessage];
 
-    return [NSString stringWithFormat:@"%@ [%@] %@", timestamp, queueThreadLabel, logMessage->_message];
+    return [NSString stringWithFormat:@"%@ [%@ (QOS:%s)] %@", timestamp, queueThreadLabel, _qos_string(logMessage->_qos), logMessage->_message];
```

Comment:
Could person opt-out of these "qos-codes"? ( for example, somebody may have parsers for logs... )

Reply:
You shouldn't parse logs that aren't formatted by your code. We don't guarantee a steady log format (neither does Apple btw)

Reply:
@ffried Sad :(

Reply:
We could add options property for DDDispatchQueueLogFormatter class. And default it by not printing QOS…

Reply:
What we should definitively do is to not print it if it's not available.
But otherwise I'd have no problem with simply adding it. Like I said, Apple does the same.
You can always run your own formatter that does not print it. Especially since we don't use this formatter by default.

Reply:
See 3a35cc7